### ISSUE TITLE
Explain the value of signing up

### ISSUE THREAD

**Issue by mhucka** [datatogether/learning#21]
The gitbook lesson we're putting together suggests that people can optionally register on Data Together, but doesn't explain why they would want to do that. What are the benefits of signing up?

**Comment by dcwalk** [comment]
Agreed @mhucka -- @b5 has some language around here. I think we've marked as optional, which is nice, but some clean up to indicate why.

Currently I don't think it is clearly enumerated anywhere the difference between logged in/not (which is likely beyond the scope of this repo)... again I'm wondering if we need bits of this convo in https://github.com/datatogether/datatogether

**Comment by mhucka** [comment]
Yeah, I agree. For the tutorial(s), I think it'd be enough to have a one sentence summary of a benefit to people to sign up.

**Comment by ebenp** [comment]
@b5 had a short sentence that he said to our tutorial that I thought captured the reasons for signing up well. I can't remember it though....

**Comment by ebarry** [comment]
you have to be signed up to add metadata

--

+1 336-269-1539 / @lizbarry <http://twitter.com/lizbarry> / lizbarry.net
